components/transferForm.tsx

```diff
@@ -827,6 +846,21 @@ function TransferForm(
                                         rows={10}
                                       />
                                     ) : (
+                                      // ) : value.kind === "autocomplete" ? (
```

Comment:
does this still use?

Reply:
Autocomplete is the first component i wanted to use for searching fa2 tokens, but as I changed it, it's somehow useless now